jupyter-tensorflow-notebook/Dockerfile

```diff
@@ -1,24 +1,6 @@
 
 FROM jupyter/tensorflow-notebook:ubuntu-22.04
 
-# Switch to root to install packages
-USER root
-
-RUN apt-get update --yes \
-    && apt-get install -y --no-install-recommends \
-    nvidia-headless-515 \
-    nvidia-utils-515 \
-    libnvidia-gl-515-server \
-    libopengl0 \
-    && apt-get install --yes \
-    curl \
-    wget \
-    patch  \
-    nvidia-cuda-toolkit \
-    && rm -rf /var/lib/apt/lists/*
```

Comment:
I can't see any of the upstream docker containers install the `nvidia` or `opengl` packages - are they not needed anymore?

Reply:
@meoflynn informs me that these are not needed